### ISSUE TITLE
`--fail-under` not working with float

### ISSUE THREAD

**Issue by epassaro** [econchick/interrogate#114]
### Environment

- `interrogate` version(s) (`interrogate --version`: 1.5.0
- Operating System(s): Linux
- Python version(s): 3.8

### Description of the bug

Passing a float to `--fail-under` flag results in fail despite the coverage is exactly the same.
```
---------------------------------------------------------------- RESULT: FAILED (minimum: 50.8%, actual: 50.8%) ----------------------------------------------------------------
```

### What you expected to happen

Expected `PASSED`.

### How to reproduce (as minimally and precisely as possible)

Get the coverage of your code, then run again with `--fail-under` and pass the exact number.

**Comment by econchick** [comment]
Just wanted to let you know that I have a fix, and will be released in the next release (`22.0` - switching to calendar versioning), hopefully by the end of the month - I'll update the issue when it's out. 

Thank you for your patience!

**Comment by epassaro** [comment]
Thanks!
